.github/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
-    "extends": "../tsconfig.json",
```

Comment:
Is this change necessary? I think we do want the GitHub Actions TS config to extend the base TS config of the project